index.html

```diff
@@ -0,0 +1,108 @@
+                    </p>
+                </div>
+                <div class="descrip">
+                    <h2>Fast Shipping</h2>
+                    <p>
+                        We make sure you recieve your trombone as soon as we have
+                        finished making it. We also provide free returns if you are not
+                        satisfied.
+                    </p>
+                </div>
+                <div class="descrip">
+                    <h2>Quality Assurance</h2>
+                    <p>
+                        For every purchase you make, we will ensure there are no damages
+                        or faults and we will check and test the pitch of your
+                        instrument.
+                    </p>
+                </div>
+            </section>
+            <section id="how-it-works">
+                <iframe
+                    id="video"
+                    height="315"
+                    src="https://www.youtube-nocookie.com/embed/y8Yv4pnO7qc?rel=0&amp;controls=0&amp;showinfo=0"
+                    frameborder="0"
+                    allowfullscreen
+                ></iframe>
+            </section>
+            <section id="pricing">
+                <article class="products" id="tenor">
+                    <div class="level">Tenor Trombone</div>
+                    <h2>$600</h2>
+                    <ol>
+                    <li>Lorem ipsum.</li>
+                    <li>Lorem ipsum.</li>
+                    <li>Lorem ipsum dolor.</li>
+                    <li>Lorem ipsum.</li>
+                    </ol>
+                    <button class="btn">Select</button>
+                </article>
+                <article class="products" id="bass">
+                    <div class="level">Bass Trombone</div>
+                    <h2>$900</h2>
+                    <ol>
+                    <li>Lorem ipsum.</li>
+                    <li>Lorem ipsum.</li>
+                    <li>Lorem ipsum dolor.</li>
+                    <li>Lorem ipsum.</li>
+                    </ol>
+                    <button class="btn">Select</button>
+                </article>
+                <article class="products" id="valve">
+                    <div class="level">Valve Trombone</div>
+                    <h2>$1200</h2>
+                    <ol>
+                    <li>Plays similar to a Trumpet</li>
+                    <li>Great for Jazz Bands</li>
+                    <li>Lorem ipsum dolor.</li>
+                    <li>Lorem ipsum.</li>
+                    </ol>
+                    <button class="btn">Select</button>
+                </article>
+            </section>
+        </section>      
+        <footer class="container">
+            <span>Copyright 2016, Original Trombones</span>
+            <ul>
+                <li><a href="#">Privacy</a></li>
+                <li><a href="#">Terms</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
```

Comment:
According to the requirements, <span>copyright....</span> should got after <ul>.....</ul> tag.